### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mattallen37**

### CODE REVIEW

**Comment by mattallen37** [comment]
Using the central OpenOCD repo, and supporting older RPi models (A, B, A+, B+, Z, etc.) that previously were not able to flash the firmware.

**Comment by CleoQc** [comment]
Model Z?

File: Install/update_gopigo3.sh [hunk 1]
```diff
@@ -266,8 +266,9 @@ install_python_pkgs_and_dependencies() {
     cp $GOPIGO3_DIR/Software/Python/Examples/Control_Panel/gopigo3_control_panel.desktop $PIHOME/Desktop/gopigo3_control_panel.desktop
   fi
 
-  # install the pre-compiled OpenOCD packaged with GoPiGo3
-  sudo bash $GOPIGO3_DIR/Firmware/openocd/install_openocd_compiled.sh
+  # install openocd
+  echo "Installing OpenOCD for GoPiGo3"
+  curl --silent https://raw.githubusercontent.com/DexterInd/openocd/master/openocd_install.sh | bash
```
Comment (CleoQc) [inline Install/update_gopigo3.sh]: I believe this script file gets called when doing a DexterOS_update, and thus the curl would fail on DexterOS.

**Comment by CleoQc** [comment]
Ok right. We decoupled the update files for GPG. Good thing then :)

**Comment by mattallen37** [comment]
Model 'Z', i.e. Zero. Same chip type as the B+.

**Pull request merged**